packages/zql/src/query/query-impl.ts

```diff
@@ -147,13 +147,13 @@ export abstract class AbstractQuery<
   TReturn = PullRow<TTable, TSchema>,
 > implements Query<TSchema, TTable, TReturn>
 {
-  readonly #schema: TSchema;
-  readonly #tableName: TTable;
-  readonly #ast: AST;
+  protected readonly _schema: TSchema;
```

Comment:
All these protected properties is a code smell... Maybe we should rethink how the query implementations are done?